argparse/argparse.py

```diff
@@ -140,11 +151,9 @@ def parse_args(self, args=None):
 
     def _parse_args(self, args):
         # add optional args with defaults
-        arg_dest = []
```

Comment:
I see.

I think it would be cleaner to have _Arg() take a list of alternative names for each option instead of creating a new _Arg() for each one.

The way you have it now, the usage() string prints multiple lines (and multiple descriptions) for options with short and long versions.